code/game/objects/effects/overlays.dm

```diff
@@ -485,6 +485,31 @@
 	duration = 8
 	randomdir = 0
 
+/obj/effect/overlay/telesci
```

Comment:
For some reason i thought that the duration had to be tied to the animation time, guess i'm just dumb